### DISCUSSION TITLE
conda not recognized

### DISCUSSION THREAD

**Discussion by jonathanariel** [teamalgoritma/community#230] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/107382811/181416087-2b694eaa-d93c-49e3-8501-e58d6e383ccb.png)
permisi, saya mau nanya lagi ini kenapa conda terus terusan not recognized ya? interpreter sudah tepat, sudah pakai command prompt juga

**Comment by fafilia** [comment, marked as answer]
Halo @jonathanariel , sudah coba untuk close semua terminal yang sedang terbuka?
<img width="797" alt="Capture" src="https://user-images.githubusercontent.com/14083457/182086044-6148de9c-fc81-4ef1-9a68-3402f974a58c.PNG">

Setelah itu coba untuk membuka terminal baru atau coba untuk menutup VScode dan membukanya kembali?
Sebagai tambahan apabila mungkin belum bisa juga, bisa melakukan pengaturan seperti yang tertera pada link berikut:
https://stackoverflow.com/questions/54828713/working-with-anaconda-in-visual-studio-code

**Comment by NabiilahArdini** [comment]
Halo @jonathanariel , untuk menjaga agar forum diskusi tetap berjalan kondusif, kami izin melakukan close pada diskusi ini karena belum ada respon setelah 7 hari respon terakhir. Semoga dapat dipahami.
